finngen/1_make_finngen_finemapping_manifest.py

```diff
@@ -7,79 +7,35 @@
 #
 
 
-import sys
 import os
-import pandas as pd
 import json
-import subprocess as sp
 
 def main():
 
     #
     # Args --------------------------------------------------------------------
     #
-
-    study_prefix = 'FINNGEN_R5_'
-    # Manifest files from Finngen R5
-    in_finngen = 'inputs/r5_finngen.json'
-    in_snp_path_list = 'inputs/input_paths_finngen.txt'
+    input_files = os.listdir('/finemapping/input/FINNGEN/')
 
     # Path to write main manifest file
-    out_manifest = 'finngen.manifest.json'
+    out_manifest = '/finemapping/configs/finngen.manifest.json'
 
     # Output directory for individual study fine-mapping results
-    output_path = 'output/'
-
-    keep_columns = [
```

Comment:
Is this logic removed based on the format of the FinnGen studies that we were provided with? Those may have been processed for us (since we asked for a specific format) while EBI could have worked with unprocessed files. If that's the case, then it would probably be easier to not request format changes and work with original format as well. We should clarify that.

Reply:
This "old" logic is based on the finngen manifest and some meta_data. This determines what finngen studies will be processed. I have set this up to process all finngen studies that are present in the pipeline run, which made more sense.  Also we really don't need `trait`, `trait_category`, `n_cases`, `n_controls` to format the top_loci and credset files, we only need `code`.

Reply:
Ok, sounds good.